src/styles/blocks/page.scss

```diff
@@ -0,0 +1,9 @@
+.page {
+  font-family: "Roboto", sans-serif;
```

Comment:
If you set font-family here so why do you need it in goodsCard__name ?

Reply:
yes, remove it from there